source/fundamentals/stable-api.txt

```diff
@@ -0,0 +1,150 @@
+.. _golang-stable-api:
+
+==============
+{+stable-api+}
+==============
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. note::
+
+   The {+stable-api+} feature requires MongoDB Server 5.0 or later.
+
+   You should only use the {+stable-api+} feature if all the MongoDB
+   servers you are connecting to support this feature.
+
+Overview
+--------
+
+In this guide, you can learn how to specify the **{+stable-api+}** when
+connecting to a MongoDB instance or replica set. 
+
+The {+stable-api+} feature forces the server to run operations with
+behaviors compatible with the **API version** you specify. An API
```

Comment:
**Question**: does this need to be bold?
```suggestion
behaviors compatible with the API version you specify. An API
```

Reply:
In our docs, we typically bold a term we are about to define in the very next sentence or two.